src/backend/language.cpp

```diff
@@ -11,23 +11,44 @@
 **********/
 
 #include "language.h"
+#include <iostream>
 
 #include <QHash>
 #include <QLocale>
 #include <QObject>
 #include <QtGlobal>
+#include <QIODevice>
+#include <QStringLiteral>
+#include <QDir>
+#include <QFile>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QJsonArray>
+#include <QJsonValue>
+#include <QDebug>
 #include <utility>
 #include "../util/btassert.h"
 #include "managers/btlocalemgr.h"
 
 // Sword includes:
 #include <swlocale.h>
 
+// iso-codes includes
+#include <libintl.h>
+
 
 namespace {
 
+QString locale = "";
+
+
 struct LanguageMap: QHash<QString, std::shared_ptr<Language const>> {
     LanguageMap();
+    void addLanguage(QStringList abbrevs, QString englishName);
+    void InitGetText(void);
+    void LoadISOCodes(void);
+
+    const char *localeDir;
```

Comment:
In BibleTime we usually have `smallCamelCase` for function names and `m_smallCamelCase` for members.